### PR TITLE
grammarly

### PR DESCRIPTION
**Pull request by daallgeier**

### CODE REVIEW

File: _data/vendors.yml [hunk 1]
```diff
@@ -158,6 +158,16 @@
   pricing_source: https://github.com/pricing
   updated_at: 2020-04-14
 
+- name: Grammarly
+  url: https://www.grammarly.com
+  base_pricing: $12.50 per u/m
+  sso_pricing: $583.33 per u/m [^grammarly]
+  percent_increase: 5000%
```
Comment (mbainter) [inline _data/vendors.yml:165]: Should be 4567%

**Review by mbainter — CHANGES_REQUESTED** [review]

File: _data/vendors.yml (hunk 1 above)
Comment (daallgeier) [reply _data/vendors.yml]: maths is hard lol

Comment (mbainter) [reply _data/vendors.yml]: We really should add the Contributors.md from the parent repository and maybe a link to [something like this](https://www.omnicalculator.com/math/percentage-increase) to make it easier for submitters.

**Review by mbainter — APPROVED** [review]

File: _data/vendors.yml (hunk 1 above)
Comment (daallgeier) [reply _data/vendors.yml]: true. 
i do not code though. so i can't built anything like that.

Comment (daallgeier) [reply _data/vendors.yml]: but let's move this to somewhere else #10, maybe?

**Pull request merged**